Sasquatch/Sasquatch/ViewControllers/MSAnalyticsViewController.swift

```diff
@@ -1,16 +1,28 @@
 import UIKit
 
 class MSAnalyticsViewController: UITableViewController, AppCenterProtocol {
+  
+  enum Priority: String {
+    case Default = "Default"
```

Comment:
Can you add Normal and Invalid as well? Android test app has those options, too.